### DISCUSSION TITLE
Will it run as nonroot user if the pod security policy is "mustRunAsNonRoot" in K8s or GKE Cluster

### DISCUSSION THREAD

**Discussion by Deepak-Suresh** [drakkan/sftpgo#585] [Q&A] [answered]
I ahve pod security policy defined as "mustRunAsNonRoot" . SO All the services running inside the container must run as non root user.  is it possible in this?

**Comment by drakkan** [comment, marked as answer]
Hi,

SFTPGo images run using 1000 as UID/GID by default. Please refer to the [docs](https://github.com/drakkan/sftpgo/tree/main/docker#running-as-an-arbitrary-user) if you want to run as another user
